adagrams/game.py

```diff
@@ -1,11 +1,181 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+    letter_pool_list = ('A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
```

Comment:
I loved how you used a tuple to store your data! There is really only one draw back to this and that it is hard coded. We usually want to avoid hard coding variables and values inside of our code base because it can become a bit of a pain to change we need to. So, keeping that in mind here's an alternative approach that follows that: 

```python
LETTER_POOL = {
    'A': 9, 
    'B': 2, 
    'C': 2, 
    'D': 4, 
    'E': 12, 
    'F': 2, 
    'G': 3, 
    'H': 2, 
    'I': 9, 
    'J': 1, 
    'K': 1, 
    'L': 4, 
    'M': 2, 
    'N': 6, 
    'O': 8, 
    'P': 2, 
    'Q': 1, 
    'R': 6, 
    'S': 4, 
    'T': 6, 
    'U': 4, 
    'V': 2, 
    'W': 2, 
    'X': 1, 
    'Y': 2, 
    'Z': 1
}

def draw_letters():
   letters = []
   drawn_letters = []
   for letter in LETTER_POOL.keys():
       letters += [letter * LETTER_POOL[letter]

   while True:
    index = random.randint(0,len(letters) - 1)
    chosen_letter = letters[index]
    if len(drawn_letters) == 10: break

    if drawn_letters.count(chosen_letter) < LETTER_POOL[chosen_letter]:
        drawn_letters.append(chosen_letter)
    else:
        continue
   return drawn_letters

```

Basically, we are looping through building a list of letters with their appropriate weights, respectively.